sw/device/silicon_creator/lib/drivers/flash_ctrl.h

```diff
@@ -494,27 +499,26 @@ void flash_ctrl_info_perms_set(const flash_ctrl_info_page_t *info_page,
  * `kMultiBitBool4True` to enable and `kMultiBitBool4False` to disable
  * these settings.
  *
- * This struct has no padding, so it is safe to `memcmp()` without invoking UB.
+ * The bitfields in this stuct match up with the bitfields in the peripheral
+ * register.
  */
 typedef struct flash_ctrl_cfg {
+  uint32_t _pad0 : 16;
   /**
    * Scrambling.
    */
-  multi_bit_bool_t scrambling;
+  uint32_t scrambling : 4;
   /**
-   * ECC.
+   * High endurance.
```

Comment:
The stringdocs need to be updated to match the fields. // ECC --> ecc, // High endurance --> he

Reply:
Fixed.